docs/WindowsBuild.md

```diff
@@ -117,23 +125,23 @@ cmake --build "S:\build\Ninja-DebugAssert\cmark-windows-amd64"
 ```cmd
 mkdir "S:\build\Ninja-DebugAssert\llvm-windows-amd64"
 pushd "S:\build\Ninja-DebugAssert\llvm-windows-amd64"
-cmake -G "Ninja"^
+cmake -G Ninja^
  -DCMAKE_BUILD_TYPE=Debug^
  -DCMAKE_C_COMPILER=cl^
  -DCMAKE_CXX_COMPILER=cl^
  -DLLVM_DEFAULT_TARGET_TRIPLE=x86_64-unknown-windows-msvc^
- -DLLVM_ENABLE_ASSERTIONS=TRUE^
+ -DLLVM_ENABLE_ASSERTIONS=ON^
```

Comment:
The reason for using `TRUE` or `YES` here is so it sounds more like English.  `YES`, `TRUE`, and `ON` are synonyms for CMake.